napari/layers/base/base.py

```diff
@@ -934,6 +939,8 @@ def _slice_dims(self, point=None, ndisplay=2, order=None):
         ----------
         point : list
             Values of data to slice at in world coordinates.
+        thickness: list
+            Thickess of slices for each dimensions in world coordinates.
```

Comment:
Sounds like we are in world coordinates here, which makes sense

Reply:
Actually, this docstring is outdated. Right now it's like this:
- dims has thickness in steps
- dims passes slice thickness and the world space size of steps to layer
- layer calculates thickness in data coords for itself based on the above

Reply:
yeah, something like that makes sense